fluid-project/trivet
# feat: basic Eleventy 1.0.0 compatibility (resolves #194)

## Description

This PR updates some dependencies, including updating Eleventy to 1.0. One minor breaking change related to liquid templating was resolved. Additionally, an explicit dependency was added for @babel/core to resolve #217.

## Steps to test

1. Check out this branch.
2. Install dependencies: `npm ci`.
3. Run build: `npm run build`.
4. Run start: `npm start`.
5. Run lint: `npm run lint`.

**Expected behavior:** Start and build commands work as expected. Linting passes.

## Additional information

Not applicable.

## Related issues

- Resolves #194.
- Resolves #217.
